Execution using `testkomodo.sh` yield incomplete information when missing project subsystem

I wanted to run komodo workflow `test_run_one_project` and test the failure experienced in the nightly komodo bleeding build.
I ran the workflow using project `ert`, but the correct option I wanted to test was `ert[ert]`.

Ref.
https://github.com/equinor/komodo-releases/actions/runs/16823435048

```
+ pushd /tmp/f_scout_ci/actions-runner-01/_temp/test_root
/tmp/f_scout_ci/actions-runner-01/_temp/test_root /tmp/f_scout_ci/actions-runner-01/_temp/source_root /tmp/f_scout_ci/actions-runner-01/_temp /tmp/f_scout_ci/actions-runner-01/komodo-releases/komodo-releases
+ start_tests
+ export NO_PROXY=localhost,127.0.0.1
+ NO_PROXY=localhost,127.0.0.1
+ export ERT_PYTEST_ARGS=--eclipse-simulator
+ ERT_PYTEST_ARGS=--eclipse-simulator
+ pushd /tmp/f_scout_ci/actions-runner-01/_temp/test_root/tests/ert
+ '[' '' == ert ']'
+ '[' '' == everest ']'
+ '[' '' == everest-egg ']'
+ '[' '' == ert-limit-memory ']'
+ '[' '' == ert-queue-system ']'
+ '[' '' == opm-integration ']'
+ echo 'Error: Variable  did not match any testable subsystem'
+ return 1
/tmp/f_scout_ci/actions-runner-01/_temp/test_root/tests/ert /tmp/f_scout_ci/actions-runner-01/_temp/test_root /tmp/f_scout_ci/actions-runner-01/_temp/source_root /tmp/f_scout_ci/actions-runner-01/_temp /tmp/f_scout_ci/actions-runner-01/komodo-releases/komodo-releases
Error: Variable  did not match any testable subsystem
Error: Process completed with exit code 1.
```

https://github.com/equinor/ert/blob/11358466649707ecf4ffa8ff91892561cd1a8aa6/ci/testkomodo.sh#L127

Perhaps we should list the possible options instead of printing an empty variable? That would also cover examples that did not match any of the valid options.

--- a/ci/testkomodo.sh
+++ b/ci/testkomodo.sh
@@ -124,6 +124,12 @@ start_tests() {
       return $?
     fi
 
-    echo "Error: Variable $CI_SUBSYSTEM_TEST did not match any testable subsystem"
+    if [ -n "$CI_SUBSYSTEM_TEST" ]; then
+      echo "Error: No argument for subsystem was provided."
+      echo "Possible subsystems are specified with ert[<subsystem>]."
+    else
+      echo "Error: Variable $CI_SUBSYSTEM_TEST did not match any testable subsystem"
+    fi
+    echo "Possible subsystems are: ert, everest, everest-egg, ert-limit-memory, ert-queue-system, opm-integration"
     return 1
 }

